services/install-pylon.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/pylon
+
+  # create yml(s)
+  {
+echo "version: 2.1"
+echo "services:"
+echo "  pylon:"
+echo "    image: linuxserver/pylon"
+echo "    environment:"
+echo "      - PYUSER=\${PYUSER}"
+echo "      - PYPASS=\${PYPASS}"
+echo "    volumes:"
+echo "      - \"/srv/pylon:/root/.pylon\""
+echo "      - /var/run/docker.sock:/var/run/docker.sock"
+echo "    ports:"
+echo "      - 3131:3131"
+echo "version: \"2.1\""
+  } > /srv/pylon/pylon.yml
+
+  # create .env with default values
+  {
+    echo "PYUSER=root"
+    echo "PYPASS=root"
+  } > /srv/pylon/.env
+  
+  # add autorun
+  {
+    echo "pylon_autorun=true"
+    echo
+    echo "if [ \"\$pylon_autorun\" = true ]; then"
+    echo "  treehouses services pylon up"
+    echo "fi"
+    echo
+    echo
+  } > /srv/pylon/autorun
+}
+
+# environment var
+function uses_env {
+  echo true
+}
+
+# add supported arm(s)
+function supported_arms {
+  echo "armv7l"
+  echo "aarch64"
+  echo "x86_64"
+}
+
+# add port(s)
+function get_ports {
+  echo "3131"
+}
+
+ #add size (in MB)
+function get_size {
+ echo "255"
+}
+
+# add info
+function get_info {
+  echo "https://github.com/linuxserver/docker-pylon.git"
+  echo
+  echo "\"Pylon is a web based integrated development environment built with Node.js as a backend and with a supercharged JavaScript/HTML5 frontend\""
+}
+
+# add svg icon
+function get_icon {
```

Comment:
I don't think this is the correct icon

Reply:
@kjong can we go with original cloud9 icon? any suggestion?

Reply:
I'm not sure if it's too important but the current icon you have is for something different from the IDE.